sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Metrics/ExecutableLinesWalker.cs

```diff
@@ -29,16 +32,29 @@ public class ExecutableLinesWalker : CSharpSyntaxWalker
     {
         private readonly HashSet<int> executableLineNumbers = new HashSet<int>();
 
+        private static readonly string[] ExcludeAttrs =

```

Comment:
Don't we have these attributes also defined somewhere else? If so it is probably better to reuse rather than copying strings.

Reply:
We don't (I was surprised myself). We had them in auto-generated code detector, but I've removed them from there.